build.gradle.kts

```diff
@@ -98,6 +98,9 @@ configure(subprojects.filterNot { it in internalBomModules }) {
         testImplementation("org.springframework.boot:spring-boot-starter-test") {
             exclude(group = "org.junit.vintage", module = "junit-vintage-engine")
         }
+
+
+        implementation("org.jetbrains:annotations:26.0.2")
```

Comment:
nit: `graphql-dgs-client/build.gradle.kts` uses `org.jetbrains:annotations:26.0.1`, not a big deal but would be better to have all aligned (ideally some BOM should just recommend it for the repo)